build-test.cmd

```diff
@@ -265,7 +269,38 @@ if errorlevel 1 (
     exit /b 1
 )
 
+REM Prepare the Test Drop
+REM Cleans any NI from the last run
+powershell "Get-ChildItem -path %__TestWorkingDir% -Include '*.ni.*' -Recurse -Force | Remove-Item -force"
```

Comment:
@wtgodbe I think incremental test runs are broken now. In a typical dev workflow. You build the tests once and build product repeatedly. Previously we used to sanitize the test build by removing test runtime artifacts like
- *.ni.*
- lock folder
- the shared framework [ including coreclr and associated stuff]

Now by moving this logic into buildtest.cmd, you have created an additional step, hopefully this cleanup can occur without rebuilding the test, if so at minimum please update the steps in https://github.com/dotnet/coreclr/blob/master/Documentation/building/windows-test-instructions.md